packages/tree-widget/src/components/TreeUiItemsProvider.tsx

```diff
@@ -0,0 +1,127 @@
+import { I18N } from "@bentley/imodeljs-i18n";
+import {
+  AbstractWidgetProps,
+  StagePanelLocation,
+  StagePanelSection,
+  UiItemsProvider,
+} from "@bentley/ui-abstract";
+import {
+  UiFramework,
+  SpatialContainmentTree,
+  ClassGroupingOption
+} from "@bentley/ui-framework";
+import React from "react";
+import { TreeWidgetComponent } from "./TreeWidgetComponent";
+import { CategoriesTreeComponent, ModelsTreeComponent, TreeWidgetControlOptions } from "../tree-widget-react";
+import { IModelConnection, Viewport } from "@bentley/imodeljs-frontend";
+import { SelectableContentDefinition } from "@bentley/ui-components";
+
+/** Provides the property grid widget to zone 9 */
```

Comment:
Comment is irrelevant here, not property-grid widget, and there is non zone 9 in ui-2.0